py/README.rst

```diff
@@ -103,10 +103,8 @@ Run IPython with PySparkling
 Use PySparkling as Spark Package
 --------------------------------
 .. code-block:: bash
-
-	$SPARK_HOME/bin/spark-submit
-	--packages ai.h2o:sparkling-water-core_2.10:1.6.1
-	--py-files $SPARKLING_HOME/py/dist/pySparkling-1.6.1-py2.7.egg  ./py/examples/scripts/ChicagoCrimeDemo.py
+    export SPARKLING_EGG=$(ls $SPARKLING_HOME/py/dist/)
```

Comment:
`ls $SPARKLING_HOME/py/dist/` or `ls $SPARKLING_HOME/py/dist/*.egg` - what if the directory contains some garbage files